### PR TITLE
fix setLevel

### PR DESCRIPTION
**Pull request by jquense**

### CODE REVIEW

**Review by taion — APPROVED** [review]
what

File: index.js [hunk 1]
```diff
@@ -48,7 +50,7 @@ function add(id, label = id) {
   container.add(id, {
     level: LEVEL,
     format: defaultFormat(label),
-    transports: [new ConsoleTransport({ silent: TEST })],
+    transports: [consoleTransport],
```
Comment (taion) [inline index.js:53]: so it's ok to have the same transport for each container?

**Review by taion — APPROVED** [review]
what

File: index.js (hunk 1 above)
Comment (jquense) [reply index.js:53]: yeah, according to the issues i dug up /shrug hoping v3 eventually adds a better api here.

**Pull request merged**